fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,11 +1,23 @@
-OpenTracks is a sport tracking application that completely respects your privacy.
-
-Features:
-* Sport tracking: track your sport and outdoor activities
-* Photos and markers: mark interesting locations while tracking
-* Altitude gain and loss: via barometric sensor
-* Altitude shown in EGM2008 (above mean sea level); exported as WGS84
-* Voice announcements
-* Supports Bluetooth LE sensors: heart rate, speed and distance (cycling), cadence (cycling), and power meter (cycling)
-* Export data: export tracks either as KMZ (incl. photos), KML, or GPX
-* No Internet access: Internet is not used
+A sport tracking buddy that respects your privacy.
+
+sport and outdoor activities grants happiness, skill and confidence.
```

Comment:
This paragraph must start with capitalized S leeter, right?

Reply:
Good catch :)